lib/decimal.ex

```diff
@@ -62,10 +85,18 @@ defmodule Decimal do
                     :half_down |
                     :up
 
+  @typedoc """
+  * `coef` - The coefficient of the power of `10`.
+  * `exp` - The exponent of the power of `10`.
+  * `sign` - `1` for positive. `-1` for negative.
+
+  This implementation models the `sign` as `1` or `-1` such that the complete number will be: `sign * coef * 10^exp`.
```

Comment:
We don't have to explain signs since we do it above. Move this sentence to the beginning of the typedoc.